### PR TITLE
fix/translation affordance more menu

### PR DESCRIPTION
**Pull request by matthewbcool**
This PR makes minor edits to the more menu for clarity and simplification across translations.

- Removes parenthesis to avoid confusion across languages
- 'You' provides arguably little value and makes right-left scripts more difficult to accommodate
- Keep email mask to protect email addresses in screen sharing 
- not signed in message 'you are not signed in' replaces 'you' for clarity

Before
unknown
After
Screen Shot 2022-01-24 at 2 48 51 PM

Suggesting this change following discussion in #dev-design here: https://discord.com/channels/498741086295031808/753725407957876777/931145395793449011

### CODE REVIEW

**Comment by matthewbcool** [comment]
Note the French translation is the only one that includes the strings referenced in this PR for now.  @vincentfretin Is `Vous n'êtes pas connecté` acceptable for `you are not signed in`?

**Comment by vincentfretin** [comment]
Yes, perfect.

File: src/assets/locales/fr.json [hunk 1]
```diff
@@ -310,8 +310,8 @@
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "more-menu.you": "Vous",
-  "more-menu.you-signed-in-as": "Vous (connecté(e) en tant que {email})",
+  "more-menu.not-signed-in": "Vous n'êtes pas connecté",
```
Comment (vincentfretin) [inline src/assets/locales/fr.json:313]: Ah I forgot, to be inclusive, "Vous n'êtes pas connecté(e)."

**Review by vincentfretin — CHANGES_REQUESTED** [review]

File: src/assets/locales/fr.json (hunk 1 above)
Comment (matthewbcool) [reply src/assets/locales/fr.json]: Thanks so much for your help, updated.

**Review by vincentfretin — APPROVED** [review]

**Pull request merged**